drivers/flash/flash_mspi_atxp032.c

```diff
@@ -668,7 +668,8 @@ static int flash_mspi_atxp032_init(const struct device *flash)
 		return -EIO;
 	}
 
-	if (mspi_dev_config(cfg->bus, &cfg->dev_id, MSPI_DEVICE_CONFIG_ALL, &cfg->tar_dev_cfg)) {
+	if (mspi_dev_config(cfg->bus, &cfg->dev_id, MSPI_DEVICE_CONFIG_IO_MODE,
```

Comment:
Make sure this is also reverted when you are done. There should be no change necessary.